web/src/settings_notifications.ts

```diff
@@ -11,19 +13,23 @@ import * as message_notifications from "./message_notifications";
 import {page_params} from "./page_params";
 import * as settings_components from "./settings_components";
 import * as settings_config from "./settings_config";
+import type {SettingsPanel} from "./settings_preferences";
 import * as settings_ui from "./settings_ui";
 import {realm} from "./state_data";
 import * as stream_data from "./stream_data";
 import * as stream_settings_api from "./stream_settings_api";
 import * as stream_settings_data from "./stream_settings_data";
+import {stream_specific_notification_settings_schema} from "./stream_types";
 import * as sub_store from "./sub_store";
+import type {StreamSubscription} from "./sub_store";
 import * as ui_util from "./ui_util";
 import * as unread_ui from "./unread_ui";
-import {user_settings} from "./user_settings";
+import * as user_settings from "./user_settings";
```

Comment:
I’m not convinced this wildcard import change was a successful experiment. This module’s export names were designed to make sense in an unqualified context, and `user_settings.user_settings[setting_name]` is quite a mouthful. @timabbott thoughts?

Reply:
I'll wait for @timabbott to reply to this before making the other changes you suggested

Reply:
Yeah this seems like a mistake; I did not expect a `user_settings.user_settings` to be a thing.

Reply:
Switched back to importing things individually from `user_settings`